packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-linking-step/wallet-linking-step.tsx

```diff
@@ -51,8 +55,52 @@ const WalletLinkingStep = observer(({ data }: TWalletLinkingStep) => {
                     return (
                         <WalletLinkWrapper
                             key={wallet.wallet_details.name}
-                            account_list={wallet.account_list}
-                            wallet_details={wallet.wallet_details}
+                            has_left_fork={wallet.account_list.length > 1 || is_mobile}
+                            left={() => {
```

Comment:
@aum-deriv 
i think that we can extract Left and Rigth components to upper scope of the file for better readability